ruby_on_rails/app_initialisation.md

```diff
@@ -11,11 +11,16 @@
 * [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not.
 You can do this with `gem update rails`. Beware of beta versions.
 
-* Start a new Rails project using `rails new [project-name] --database=postgresql --no-skip-test --skip-sprockets --webpack --skip_action_mailbox` where the `project-name` is exactly the one you chose before.
-You may want to choose a different database from Postgres, but most of the time that will be your choice.
+* Start a new Rails project using `rails new [project-name] --database=postgresql --skip-test --skip-action-mailbox` where the `project-name` is exactly the one you chose before.
+
+  * You can use the `--css` flag to preconfigure your new Rails app to use a CSS processor/framework. Available options are: `tailwind, bootstrap, bulma, postcss, sass`. This will automatically install the [cssbundling-rails](https://github.com/rails/cssbundling-rails) and [jsbundling-rails](https://github.com/rails/jsbundling-rails) gems. The app then uses the the [esbuild](https://github.com/evanw/esbuild) JS bundler and minifier by default.
+
+  * Esbuild is generally a good choice, but you can also configure an alternative JS bundler using the `--javascript` flag. You have the choice between using one of the node-based bundlers: `esbuild, rollup, webpack` or you can use `importmap`, which is the default. Note that importmaps cannot be used in conjunction with the `--css` option from above.
```

Comment:
* This also seems part of the Rails guide. 
* These decisions are taken by the team anyway, this paragraph does not give guidance. I'd rather state at the very beginning that the team must decide before setting up the app if they want a js bundling and which one.
* Given that I'd even restrict these decision to always use esbuild
* Given that, I'd even restrict it more to say: "no js bundling until you really need it
* Why cannot importmaps be used with `--css`? 🤔

Reply:
- I still feel like this is a good place to mention that this is possible. Especially when the bootstrap guide is gone.
- I agree with giving more guidance and restricting it to esbuild. Also for consistency reasons I guess.
- When you use `--css` it will always default to esbuild. Even if you'd say `-c bootstrap -j importmap`